Applications/CMC/test/testCMCSingleMuscle_Thelen.cpp

```diff
@@ -35,15 +35,15 @@ void testSingleMuscle() {
     cout << "*                  testSingleMuscle_Thelen                       *" << endl;
     cout << "******************************************************************\n" << endl;
     ForwardTool forward("block_hanging_from_muscle_Setup_Forward.xml");
-    forward.setResultsDir("block_hanging_from_muscle_ForwardResults_Thelen");
+    // forward.setResultsDir("block_hanging_from_muscle_ForwardResults_Thelen");
     forward.run();
 
     CMCTool cmc("block_hanging_from_muscle_Setup_CMC.xml");
-    cmc.setResultsDir("block_hanging_from_muscle_ResultsCMC_Thelen");
+    // cmc.setResultsDir("block_hanging_from_muscle_ResultsCMC_Thelen");
     cmc.run();
 
-    Storage fwd_result("block_hanging_from_muscle_ForwardResults_Thelen/block_hanging_from_muscle_states.sto");
-    Storage cmc_result("block_hanging_from_muscle_ResultsCMC_Thelen/block_hanging_from_muscle_states.sto");
```

Comment:
What's the issue here? How come changing the results directory caused an issue? Seems preferable.